README.md

```diff
@@ -62,14 +62,6 @@ In `settings.php` add the following:
 $config['updates_log']['diff'] = TRUE;
 ```
 
-### Control the scheduling
```

Comment:
We should add info how to test 2.0.
How to force immediate output?
How to get statistical output?